docs/sources/getting-started/getting-started-influxdb.md

```diff
@@ -0,0 +1,110 @@
+
+> **Note:** Windows users might need to make additional adjustments. Look for special instructions in the InfluxData documentation and [Using Telegraf on Windows](https://www.influxdata.com/blog/using-telegraf-on-windows/) blog post.
+
+> **Note:** The regular system monitoring template in InfluxDB Cloud is not compatible with Windows. Windows users who use InfluxDB Cloud to monitor their system will need to use the [Windows System Monitoring Template](https://github.com/influxdata/community-templates/tree/master/windows_system).
+
+## Step 4. Add your InfluxDB data source to Grafana
+
+You can have more than one InfluxDB data source defined in Grafana.
+
+1. Follow the general instructions to [add a data source]({{< relref "../datasources/add-a-data-source.md" >}}).
+1. Decide if you will use InfluxQL or Flux as your query language.
+   - For InfluxQL, refer to [InfluxDB data source]({{< relref "../datasources/influxdb/_index.md" >}}) for information about specific data source fields.
+   - For Flux, refer to [Flux query language in Grafana]({{< relref "../datasources/influxdb/influxdb-flux.md" >}}) for information about specific data source fields.
+
+### InfluxDB guides
+
+InfluxDB publishes guidance for connecting different versions of their product to Grafana.
+
+- **InfluxDB OSS or Enterprise 1.8+.** To turn on Flux, refer to [Configure InfluxDB](https://docs.influxdata.com/influxdb/v1.8/administration/config/#flux-enabled-false.). Select your InfluxDB version in the upper right corner.
+- **InfluxDB OSS or Enterprise 2.x.** Refer to [Use Grafana with InfluxDB](https://docs.influxdata.com/influxdb/v2.0/tools/grafana/). Select your InfluxDB version in the upper right corner.
+-  **InfluxDB Cloud.** Refer to [Use Grafana with InfluxDB Cloud](https://docs.influxdata.com/influxdb/cloud/tools/grafana/).
+
+### Important tips
+
+- Make sure your Grafana token has read access. If it doesn't, then you'll get an authentication error and be unable to connect Grafana to InfluxDB.
+- Avoid apostrophes and other non-standard characters in bucket and token names.
+- If the text name of the organization or bucket doesn't work, then try the ID number.
+- If you change your bucket name in InfluxDB, then you must also change it in Grafana and your Telegraf .conf file as well.
+
+## Step 5. Add a query
+
+This step varies depending on the query language that you selected when you set up your data source in Grafana.
+
+### InfluxQL query language
+
+In the query editor, click **select measurement**.
+
+![InfluxQL query](/img/docs/influxdb/influxql-query-7-5.png)
+
+Grafana displays a list of possible series. Click one to select it, and Grafana graphs any available data. If there is no data to display, then try another selection or check your data source.
```

Comment:
please note that after you chose something here, you might have to choose something in the line below.
currently (based on the image) it shows `field(value)`, this means, it chose the field named "value". but, if your field has a different name, you will have to click on the `value` part, and select an option.

Reply:
Can you provide a better simple instruction, then? I just want the user to be able to get results, any results, as fast as possible.